kroxylicious-api/pom.xml

```diff
@@ -179,6 +181,40 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.github.siom79.japicmp</groupId>
+                <artifactId>japicmp-maven-plugin</artifactId>
+                <version>0.18.0</version>
+                <configuration>
+                    <oldVersion>
+                        <!-- specify the old version directly so its deterministic -->
+                        <dependency>
+                            <groupId>${project.groupId}</groupId>
+                            <artifactId>${project.artifactId}</artifactId>
+                            <version>${ApiCompatability.ReferenceVersion}</version>
+                            <type>jar</type>
+                        </dependency>
+                    </oldVersion>
+                    <parameter>
+                        <accessModifier>public</accessModifier>
```

Comment:
What's this one do? "Sets the access modifier level (public, protected, package_protected, private)." I guess it's this from the introduction: `Per default private and package protected classes and class members are not compared. If necessary, the access modifier of the classes and class members to be compared can be set to public, protected, package or private.`. What is the default? Should we check protected members too?

Reply:
IIRC I copied it from the Flink config and didn't think too hard about it. My theory being we only care about the public API for semver. However you make an interesting point about protected. I wonder if we actually use protected in the API.

IIRC the default is `public` but I would need to confirm.